nixos/modules/hardware/video/hidpi.nix

```diff
@@ -12,11 +12,11 @@ with lib;
     boot.loader.systemd-boot.consoleMode = mkDefault "1";
 
 
-    # Grayscale anti-aliasing for fonts
-    fonts.fontconfig.antialias = mkDefault true;
-    fonts.fontconfig.subpixel = {
-      rgba = mkDefault "none";
-      lcdfilter = mkDefault "none";
+    # Disable font anti-aliasing, hinting, and sub-pixel rendering by default
```

Comment:
Could I bother you with a link to fontconfig upstream recommendations?

Reply:
(as a comment of this file)

Reply:
I was referring to the comments on the `fontconfig.subpixel` options [in the NixOS manual](https://nixos.org/manual/nixos/unstable/options.html#opt-fonts.fontconfig.subpixel.lcdfilter):
- `antialias`: "At high resolution (> 200 DPI), antialiasing has no visible effect; users of such displays may want to disable this option."
- `hinting`: "[...] hinting will do nothing (at best);"
- `lcdfilter`: "LCD filtering has no visible effect; users of such displays may want to select `none`."

`fontconfig.nix` does not specify an external reference for those recommendations, and that specific wording does not come up outside NixOS in a web search, but Arch's wiki etc. have similar recommendations.

I can't find something offhand specifically from fontconfig's upstream though, their user manual just covers the configuration syntax.

Reply:
PS: Is it sufficient to link back to the manual?  At least this changes make things internally coherent

Reply:
No needed if it comes from us somewhat, I thought there was a proper fontconfig page somewhere on this, let's mark it resolved